app/models/name.rb

```diff
@@ -2287,6 +2222,129 @@ def change_deprecated(deprecated)
 
   ################################################################################
   #
+  #  :section: Merging
```

Comment:
Are there any changes in this section?  Did you just move it all verbatim?  I can do a diff by hand if there are changes, no worries, just want to know if I need to take the time.